src/coreclr/jit/fgopt.cpp

```diff
@@ -1834,7 +1834,7 @@ bool Compiler::fgCanCompactBlocks(BasicBlock* block, BasicBlock* bNext)
         return false;
     }
 
-#if defined(TARGET_ARM)
+#if defined(FEATURE_EH_FUNCLETS) && defined(TARGET_ARM)
```

Comment:
What is this trying to achieve? `FEATURE_EH_FUNCLETS` is always enabled for ARM, so this change is no-op.